CMakeLists.txt

```diff
@@ -19,7 +26,25 @@ set(SOURCE
 )
 
 add_executable(libtree ${SOURCE})
-target_link_libraries(libtree cxxopts termcolor cppglob ELFIO)
-target_include_directories(libtree PRIVATE include deps/cppglob/include ${CMAKE_CURRENT_BINARY_DIR})
+target_include_directories(libtree PRIVATE
+    include
+    ${CMAKE_CURRENT_BINARY_DIR}
+)
+
+if(USE_SYSTEM_DEPS)
+    target_link_libraries(libtree cppglob)
+else()
+    target_link_libraries(libtree cxxopts termcolor cppglob ELFIO)
+    target_include_directories(libtree PRIVATE
+        deps/cppglob/include
+        deps/termcolors/termcolors
```

Comment:
isn't it deps/termcolors/include/termcolors ?

Reply:
No.
The header its included like this `#include <termcolor/termcolor.hpp>`